paddle/fluid/operators/detail/rpc_server.h

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
I think we can now move this file under `fluid/framework` as a general interface now.